test/org/sagebionetworks/bridge/services/StudyServiceImplTest.java

```diff
@@ -141,15 +126,49 @@ public void crudStudy() {
         identifier = null;
     }
     
-    private Study createStudy() {
-        study = new DynamoStudy();
-        study.setIdentifier(identifier);
-        study.setName("Test of study creation");
-        study.setMaxNumOfParticipants(100);
-        study.setMinAgeOfConsent(18);
-        study.setResearcherRole(identifier+"_researcher");
-        study.setStormpathHref("http://dev-test-junk");
-        return study;
+    @Test
+    public void defaultsAreUsedWhenNotProvided() {
+        study = TestUtils.getValidStudy();
+        study.setPasswordPolicy(null);
+        study.setVerifyEmailTemplate(null);
+        study.setResetPasswordTemplate(null);
+        identifier = study.getIdentifier();
+        
+        study = studyService.createStudy(study);
+        
+        assertEquals(PasswordPolicy.DEFAULT_PASSWORD_POLICY, study.getPasswordPolicy());
+        assertNotNull(study.getVerifyEmailTemplate());
+        assertNotNull(study.getResetPasswordTemplate());
+
+        // Even if partial values are submitted in the JSON, we don't get exceptions, we get defaults
+        study.setVerifyEmailTemplate(new EmailTemplate(null, "body ${url}", MimeType.TEXT));
+        study.setResetPasswordTemplate(new EmailTemplate("subject", null, MimeType.TEXT));
+
+        study = studyService.updateStudy(study);
+        
+        assertEquals("Verify your account", study.getVerifyEmailTemplate().getSubject());
+        assertNotNull(study.getResetPasswordTemplate().getBody());
+        assertTrue(study.getResetPasswordTemplate().getBody().contains("To reset your password please click on this link"));
+    }
```

Comment:
Regarding migration, I'm going to NOT set defaults for one of the studies that's in the whitelist (the studies already in production and being used). These will have to explicitly set the templates or password policy, just in case someone tries to update the study before I migrate it.